common/src/main/res/drawable/ic_2_g_plus.xml

```diff
@@ -0,0 +1,14 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
File name is inconsistent with the others (here it's `ic_2_g_...` and the others are `ic_2g_...`)